cairo/src/contracts/token/components/hyp_erc20_collateral_component.cairo

```diff
@@ -80,6 +81,11 @@ pub mod HypErc20CollateralComponent {
         fn balance_of(self: @ComponentState<TContractState>, account: ContractAddress) -> u256 {
             self.wrapped_token.read().balance_of(account)
         }
+
+        fn get_wrapped_token(self: @ComponentState<TContractState>) -> ContractAddress {
+            let wrapped_token: ERC20ABIDispatcher = self.wrapped_token.read();
```

Comment:
Just a matter of form and to reduce verbosity:

```rust
fn get_wrapped_token(self: @ComponentState<TContractState>) -> ContractAddress {
            self.wrapped_token.read().contract_address
}
```

Reply:
Compiler is failing to infer type